packages/contentful--app-scripts/lib/utils.js

```diff
@@ -13,6 +14,32 @@ const throwValidationException = (subject, message, details) => {
   throw new TypeError(message);
 };
 
+const isValidIpAddress = (address) => {
+  const addressRegex =
+    /^(?:(?:https?|ftp):\/\/)?(?:localhost|(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\.)+[a-zA-Z]{2,6}|(?:\d{1,3}\.){3}\d{1,3}|(\[(?:[A-Fa-f0-9]{1,4}:){7}[A-Fa-f0-9]{1,4}\]|(?:[A-Fa-f0-9]{1,4}:){7}[A-Fa-f0-9]{1,4}))(?::\d{1,5})?$/;
```

Comment:
looks like its matching for protocols, which you remove in ln 115, so why still match for those?

Reply:
Users can still have input the domains with protocol, according to them it is correct domain but for our case we don't need it so that is why I am removing protocol from this `removeProtocolFromUrl` function.

According to the ticket I 🤔 we can accept it but we can remove it internally and then I can send it to backend

Reply:
I see, but you are removing protocol with removeProtocolFromUrl before checking with this method right? 
It's not that big of a deal, it's just that it's checking for things that are not relevant.

Reply:
Yes, make sense